opwen_email_server/jobs/store_inbound_emails.py

```diff
@@ -14,6 +15,7 @@ def _process_message(self, message: dict):
         self.log_info('Fetched inbound MIME email %s', resource_id)
 
         email = parse_mime_email(mime_email)
+        convert_img_url_to_base64(email)
```

Comment:
`convert_img_url_to_base64` returns an email, so shouldn't we use the return value here? `email = convert_img_url_to_base64(email)`